src/index.js

```diff
@@ -136,34 +124,119 @@ export async function initConversionTracking(parent = document, defaultFormConve
+        if (section.dataset.conversionValueField) {
+          const cvField = section.dataset.conversionValueField.trim();
+          // this will track the value of the element with the id specified in
+          // the "Conversion Element" field.
+          // ideally, this should not be an ID, but the case-insensitive name label of the element.
+          sampleRUM.convert(undefined, (cvParent) => findConversionValue(cvParent, cvField), element, ['submit']);
+        }
+        let formConversionName = section.dataset.conversionName || getMetadata('conversion-name');
+        if (!formConversionName) {
+          // if no conversion name is defined in the metadata,
+          // use the conversion name passed as parameter or the form or id
+          formConversionName = defaultFormConversionName
+            ? toClassName(defaultFormConversionName) : element.id;
+        }
+        sampleRUM.convert(formConversionName, undefined, element, ['submit']);
+      });
+    },
+    link: () => {
+      // track all links
+      Array.from(parent.querySelectorAll('a[href]'))
+        .map((element) => ({
+          element,
+          cevent: getConversionNameMetadata(context, element) || getMetadata('conversion-name') || getLinkLabel(context, element),
+        }))
+        .forEach(({ element, cevent }) => {
+          sampleRUM.convert(cevent, undefined, element, ['click']);
+        });
+    },
+    'labeled-link': () => {
+      // track only the links configured in the metadata
+      const linkLabels = getMetadata('conversion-link-labels') || '';
+      const trackedLabels = linkLabels.split(',')
+        .map((p) => p.trim())
+        .map(toClassName);
 
-  const declaredConversionElements = this.getMetadata('conversion-element') ? this.getMetadata('conversion-element').split(',').map((ce) => this.toClassName(ce.trim())) : [];
+      Array.from(parent.querySelectorAll('a[href]'))
+        .filter((element) => trackedLabels.includes(getLinkLabel(context, element)))
+        .map((element) => ({
+          element,
+          cevent: getConversionNameMetadata(context, element) || getMetadata('conversion-name') || getLinkLabel(context, element),
+        }))
+        .forEach(({ element, cevent }) => {
+          sampleRUM.convert(cevent, undefined, element, ['click']);
+        });
+    },
+  };
+  const declaredConversionElements = getMetadata('conversion-element') ? getMetadata('conversion-element').split(',').map((ce) => toClassName(ce.trim())) : [];
 
   Object.keys(conversionElements)
     .filter((ce) => declaredConversionElements.includes(ce))
     .forEach((cefn) => conversionElements[cefn]());
 }
+
+// Add support for Plugin system
+/**
+ * Load the martech configured as non-delayed
+ * @param {*} context should contain at lease sampleRUM object and toCamelCase function
+ */
+export async function loadLazy(document, pluginOptions, context) {
+  initCTInternal(context, document);
```

Comment:
Is it intended that we hide the `defaultFormConversionName` option now?

Reply:
`defaultFormConversionName` is intended to be used only when initializing a fragment that was added to the DOM after initial instrumentation. E.g. when loading a modal fragment with a form when user clicks a certain button

Reply:
I have updated the documentation